Ginger/GingerCoreNET/Run/RunSetActions/RunSetActionHTMLReportSendEmailOperations.cs

```diff
@@ -388,8 +388,6 @@ public void Execute(IReportInfo RI)
                 RunSetActionHTMLReportSendEmail.Email.MailCC = mValueExpression.ValueCalculated;
                 mValueExpression.Value = RunSetActionHTMLReportSendEmail.Subject;
                 RunSetActionHTMLReportSendEmail.Email.Subject = mValueExpression.ValueCalculated;
-                mValueExpression.Value = RunSetActionHTMLReportSendEmail.MailHost;
```

Comment:
why it was removed, value exression was used to make host configurable using variable

Reply:
Since the Mail Host value expression calculation is also done in EmailOperations.cs